### ISSUE TITLE
Can I show local screenshot in the report using addContext()?

### ISSUE THREAD

**Issue by mjanardhanreddy** [adamgruber/mochawesome#309]
I am trying to add the screenshot through context which exists in My Local computer, and when the report is generated I am able to see the images in the report in my local machine, when I export that report and email it to someone they are able to see the broken Images, cause being the images doesn't exist in their system......

Is there a way I can attach the images through HTML, attachments(CDN)
Please help me... @adamgruber 
Help much appreciated.

it('Take screenshot', function () {
    // context can be a url and the report will create a link
    addContext(this, {
      html: '<img src="cid:unique@kreata.ee"/>',
      attachments: [{
          filename: 'image.png',
          path: __dirname + '/screenshots/image.png',
          cid: 'unique@kreata.ee' //same cid value as in the html img src
      }]
    });
  }, 120000);

  Error adding context: Expected a string or an object of shape { title: string, value: any } but saw:
  {
    "html": "<img src=\"cid:unique@kreata.ee\"/>",
    "attachments": [
      {
        "filename": "Budgetary.png",
        "path": "/home/user/Desktop/HPE_Puppeteer/ngit_ecomm/hybris/bin/custom/hpe/pointnextstorefront/web/webroot/WEB-INF/_ui-src/react/e2e_tests/pages/screenshots/Budgetary.png",
        "cid": "unique@kreata.ee"
      }
    ]
  }

**Comment by kolbasik** [comment]
Hi @mjanardhanreddy , you could use [DataUri](https://en.wikipedia.org/wiki/Data_URI_scheme) in order to inline images directly to the report.
```js
addContext(this, {
    title: "screenshot",
    value: "data:image/png;base64," + screenshot.toString("base64")
});
```

**Comment by caidanny** [comment]
hi @mjanardhanreddy, could you share the code that converts the image to base64 to and use it in addContext, thanks in advance

**Comment by adamgruber** [comment]
Yes — point `addContext` at a reachable image. For a report you'll share or email, the most robust option is to inline the image as a [data URI](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs) (base64) so it travels with the HTML; a bare local file path only resolves on the machine that has the file (`file:///absolute/path` works locally). Closing as answered.

---
_Triaged and posted via Claude Code, on behalf of @adamgruber._
